test/junitQueryTest/JUnitQueryTest.gmk

```diff
@@ -0,0 +1,246 @@
+#
+# Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+#
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.  Oracle designates this
+# particular file as subject to the "Classpath" exception as provided
```

Comment:
The use of GPL+CPE seems unusual for a test.

Reply:
will fix

Reply:
Actually, this is a `*.gmk` file, and the file is in line with the existing practice of using GPL+CE for all makefiles, including test makefiles.